.gitpod.yml

```diff
@@ -1,5 +1,26 @@
 tasks:
-  - init: nvm install; nvm use; npm i -g npm@next-8; npm ci; npm run lerna-ci; npm run init-environment
+  - init: |
+      # Install Nodejs version specified in .nvmrc
+      nvm install
+      nvm use
+      # Install npm 8.x
+      npm i -g npm@next-8
```

Comment:
(nit) Given that we're going to be in  #877 . 
We could eventually remove this?

Reply:
Yes, I kept it mostly for consistency with the docs / manual setup and also to make it more explicit.

I checked [this table here](https://nodejs.org/en/download/releases/) and up to `Node.js 16.10.0` the default version was still `npm 7.x`, so at least for manual setup I think it's better to have it there since not everyone will be using `.nvmrc`.

What do you think?